manifests/config.pp

```diff
@@ -94,5 +94,12 @@
     file { '/var/lib/ntp/ntp.conf.dhcp':
       ensure => absent,
     }
+
+    #remove dhclient ntp script which modifies ntp.conf on RHEL and Amazon Linux
+    #backup created in case admin wants to revert back to dhclient
+    file { '/etc/dhcp/dhclient.d/ntp.sh':
+      ensure => absent,
+      backup => true,
```

Comment:
This would unexpectedly override global file bucket policy, I think this should be left up to user of the module, or make it configurable

Reply:
do you mean for creating a backup?

Reply:
Correct

Reply:
done.  Also, please bare with my many commits... first time writing testing.

Reply:
Maintainer will ask you to squash them into one, when it's finalized